src/states/main_game.rs

```diff
@@ -81,6 +83,16 @@ impl Default for MainGameState {
                     "time_control",
                     &[],
                 )
+                .with(
+                    spawner::CreatureSpawnerSystem::default(),
+                    "creature_spawner",
+                    &[],
```

Comment:
should depend on `debug_spawn_trigger`, so that events generated by the debug system will be read in the same frame

Reply:
I added the dependency